import numpy as np

<<<<<<< HEAD
np.little_endian = np.little_endian  # E: Cannot assign to final
=======
np.Inf = np.Inf  # E: Cannot assign to final
np.ALLOW_THREADS = np.ALLOW_THREADS  # E: Cannot assign to final
np.little_endian = np.little_endian  # E: Cannot assign to final
np.UFUNC_PYVALS_NAME = "bob"  # E: Incompatible types
np.CLIP = 2  # E: Incompatible types
>>>>>>> 7ef3263bf8ad3edda12247249d99ea6f271604bd
