<<<<<<< HEAD
import numpy.lib.array_utils as array_utils

array_utils.byte_bounds(1)  # E: incompatible type
=======
<<<<<<< HEAD
import numpy.lib.array_utils as array_utils

array_utils.byte_bounds(1)  # E: incompatible type
=======
import numpy as np

np.deprecate(1)  # E: No overload variant

np.deprecate_with_doc(1)  # E: incompatible type

np.byte_bounds(1)  # E: incompatible type

np.who(1)  # E: incompatible type

np.lookfor(None)  # E: incompatible type

np.safe_eval(None)  # E: incompatible type
>>>>>>> 7ef3263bf8ad3edda12247249d99ea6f271604bd
>>>>>>> f381b97f3c4908d545a6261474c75f6902f79357
