import numpy as np

<<<<<<< HEAD
np.isdtype(1, np.int64)  # E: incompatible type
=======
# Technically this works, but probably shouldn't. See
#
# https://github.com/numpy/numpy/issues/16366
#
np.maximum_sctype(1)  # E: No overload variant

np.issubsctype(1, np.int64)  # E: incompatible type
>>>>>>> 7ef3263bf8ad3edda12247249d99ea6f271604bd

np.issubdtype(1, np.int64)  # E: incompatible type
